app/src/main/java/com/fivesysdev/standwithukraine/ui/StatisticFragment.kt

```diff
@@ -107,6 +108,8 @@ class StatisticFragment : Fragment(R.layout.fragment_statistic) {
             requireContext(),
             OrientationHelper.HORIZONTAL
         )
+        getDrawable(requireContext(), R.drawable.divider)?.let { horizontalDivider.setDrawable(it) }
+        getDrawable(requireContext(), R.drawable.divider)?.let { verticalDivider.setDrawable(it) }
```

Comment:
is it possible to combine this in one block?

Reply:
Yes, it is possible. I don't know why I did it that way (maybe at that moment I was thinking that there are used different drawables). It should be like this:
`getDrawable(requireContext(), R.drawable.divider)?.let {
            horizontalDivider.setDrawable(it)
            verticalDivider.setDrawable(it)
        }`